<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConstraintChooserDialog</class>
 <widget class="QDialog" name="ConstraintChooserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Constraint Chooser Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>329</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="setupConstraintsListWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>211</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>70</y>
     <width>121</width>
     <height>162</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="createPushButton">
      <property name="text">
       <string>Create
constraint</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="acceptRadioButton">
      <property name="text">
       <string>Accept constraint</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rejectRadioButton">
      <property name="text">
       <string>Reject constraint</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="updatePushButton">
      <property name="text">
       <string>Update
constraint</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deletePushButton">
      <property name="text">
       <string>Delete
constraint</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="constraintsListWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>211</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConstraintChooserDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConstraintChooserDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
